README.md

```diff
@@ -14,25 +14,25 @@ To build from source:
 
 ### Example configuration
 
-**NOTE:** Ensure that `SDC_KEY_ID` is in MD5 format (e.g.: `ssh-keygen -l -E md5
+**NOTE:** Ensure that `TRITON_KEY_ID` is in MD5 format (e.g.: `ssh-keygen -l -E md5
 -f /Users/James/.ssh/joyent` - Mac OS X does not do this by default)
 
 ```json
 {
     "variables": {
-        "sdc_url": "{{env `SDC_URL`}}",
-        "sdc_account": "{{env `SDC_ACCOUNT`}}",
-        "sdc_key_id": "{{env `SDC_KEY_ID`}}",
-        "sdc_key_path": "{{env `SDC_KEY_PATH`}}"
+        "triton_url": "{{env `TRITON_URL`}}",
```

Comment:
This should match [`example.json`, no](https://github.com/joyent/packer-builder-triton/pull/3/files#diff-964a59d491880661889b0e908c47a1e0R3)?